src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -38,7 +33,9 @@
 #define FALLTHROUGH
 #endif
 
+#ifdef TARGET_WINDOWS
 #include <algorithm>
```

Comment:
out of curiosity, what algorithms are we using?

Reply:
It was used for min/max which I have replaced (since it was colliding with std header). I just forgot to this clean it up (TARGET_WINDOWS is a nop in this file). :)

Reply:
I'd also be fine creating a `MAX` or `MIN` macro to share. The lowercase variety are not acceptable but uppercase I can work with.